spec/spec_helper.rb

```diff
@@ -85,4 +94,25 @@
   end
 
   Sidekiq::Testing.inline!
+
+  # so we can use :vcr rather than :vcr => true;
+  # in RSpec 3 this will no longer be necessary.
+  config.treat_symbols_as_metadata_keys_with_true_values = true
+
+  # database_cleaner config from Avdi Grim (http://bit.ly/1dunbrq)
```

Comment:
This is what I would call a good comment. Full explanation by link to the documentation.